### PR TITLE
Add check on Github Status before local deploy

### PR DESCRIPTION
**Pull request by JusteLeblanc**

### CODE REVIEW

File: generators/fastlane-setup/templates/fastlane/Fastfile [hunk 1]
```diff
@@ -1,5 +1,16 @@
 require 'dotenv'
 
+# Check Git Status
+
+lane :check_git_status do |options|
+  ensure_git_branch(
```
Comment (Minishlink) [inline generators/fastlane-setup/templates/fastlane/Fastfile]: IF !ENV['CI'] then ... ?

Comment (JusteLeblanc) [inline generators/fastlane-setup/templates/fastlane/Fastfile]: J'allais le faire ;)

Comment (JusteLeblanc) [inline generators/fastlane-setup/templates/fastlane/Fastfile]: Just add it

**Comment by JusteLeblanc** [comment]
@Almouro could you check if commit message respect semantic-release convention ? Thanks 😉 
@yleflour is it OK for you ?

**Comment by Almouro** [comment]
Commit is good :)
Changes requested by @yleflour though

**Comment by Almouro** [comment]
Would need something to install modules locally though like `yarn install` or `npm install`

**Comment by Almouro** [comment]
Oups, I merged it by mistake, sorry guys :o
@JusteLeblanc can you add the changes requested by Yann with a commit name starting with `fix` or `feat`?

**Comment by JusteLeblanc** [comment]
@Almouro Je viens de faire une nouvelle PR prenant en compte de les retours de @yleflour

**Pull request merged**